ZEDCamera/Assets/package.json

```
@@ -0,0 +1,106 @@
+      "description": "Animate a 3D avatar based on real-people movements using the ZED SDK Body Tracking module.",
+      "path": "Samples~/Body Tracking"
+    },
+    {
+      "displayName": "Dark Room",
+      "description": "Your office is now a night club! Explore the lighting features coupled to the depth sensing to cast a laser show on your walls.",
+      "path": "Samples~/Dark Room"
+    },
+    {
+      "displayName": "Drone Shooter",
+      "description": "Defend yourself from drones that spawn around your room and shoot at you. Block lasers with your hand, and shoot back with the spacebar, or VR controllers if using the Oculus Integration or SteamVR plugin.",
+      "path": "Samples~/Drone Shooter"
+    },
+    {
+      "displayName": "Green Screen",
+      "description": "Aim your ZED at a greenscreen and hit Play to see your subject standing in a small town in the desert. You’ll see that the nearby crates still have all the proper occlusion, but the greenscreen background is replaced with the virtual background.",
+      "path": "Samples~/Green Screen"
+    },
+    {
+      "displayName": "Movie Screen",
+      "description": "Simple AR sample, playing a movie on a movable, scalable 3D screen integrated to the real-world. Demonstrates how 2D content can easily be displayed in a 3D, mixed reality scene.",
+      "path": "Samples~/Movie Screen"
+    },
+    {
+      "displayName": "Multicam",
+      "description": "Integrate data from multiple ZED cameras in the same scene, at the same time.",
+      "path": "Samples~/Multi Cam"
+    },
+    {
+      "displayName": "Object Detection",
+      "description": "Detect objects and people bounding boxes using the ZED SDK Object Detection module.",
+      "path": "Samples~/Object Detection"
+    },
+    {
+      "displayName": "Object Placement",
+      "description": "Place virtual objects on real-world surfaces using the Plane Detection capabilities of the ZED SDK.",
+      "path": "Samples~/Object Placement"
+    },
+    {
+      "displayName": "OpenCV ArUco Detection",
+      "description": "Print out ArUco markers, put them in view of your ZED, and let the battle of the drones begin. Shows how to easily interface the ZED with OpenCV for marker detection using a variety of included scripts.",
+      "path": "Samples~/OpenCV ArUco Detection"
+    },
+    {
+      "displayName": "Plane Detection",
+      "description": "Run the scene and hold down spacebar to see if you’re looking at a valid surface where a bunny could stand. Release the spacebar and a bunny will fall from the sky and land on that surface with proper physics.",
+      "path": "Samples~/Plane Detection"
+    },
+    {
+      "displayName": "VR Plane Detection",
+      "description": "Throw a plushie as far as possible, the distance is measured where it lands using the Plane Detection. Shows how plane detection can fit into a proper VR game.",
+      "path": "Samples~/Plane Detection"
+    },
```

```suggestion
    {
      "displayName": "Plane Detection",
      "description": "Run the scene and hold down spacebar to see if you’re looking at a valid surface where a bunny could stand. Release the spacebar and a bunny will fall from the sky and land on that surface with proper physics. In VR, throw a plushie as far as possible, the distance is measured where it lands using the Plane Detection. Shows how plane detection can fit into a proper VR game.",
      "path": "Samples~/Plane Detection"
    }
```
I suppose this is how we could merge both Plane Detection samples?